bluesky/plan_tools.py

```diff
@@ -136,21 +189,6 @@ def stage_wrapper(plan, dets):
     return ret
 
 
-def subscription_wrapper(plan, subs):
```

Comment:
I'm keeping your name, @tacaswell, but my code because I threw in more comments and a docstring. The substance is the same.